src/SFA.DAS.ApplyService.Application/Services/Assessor/AssessorLookupService.cs

```diff
@@ -76,28 +76,19 @@ public string GetTitleForPage(string pageId)
                 case RoatpWorkflowPageIds.ReadinessToEngage.SubcontractorsDueDiligence:
                     return "Due diligence on subcontractors";
 
-                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_Main:
-                    return "Type of apprenticeship training";
-                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_Employer:
-                    return "Type of apprenticeship training";
+                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_MainEmployer:
+                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_Employer_OLD_PAGE_ID:
                 case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_Supporting:
-                    return "Type of apprenticeship training";
                 case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.TypeOfApprenticeshipTraining_NewSupporting:
                     return "Type of apprenticeship training";
-                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.ApprenticeshipStandards_NewMain:
-                    return "Delivering training in apprenticeship standards";
-                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.ApprenticeshipStandards_NewEmployer:
-                    return "Delivering training in apprenticeship standards";
+                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.ApprenticeshipStandards_NewMainEmployer:
+                case RoatpWorkflowPageIds.PlanningApprenticeshipTraining.ApprenticeshipStandards_NewEmployer_OLD_PAGE_ID:
```

Comment:
see first comment.

However, I'm guessing we want to keep them here for past/ongoing cases?

Reply:
Existing applications have the old ids